doc/reference/reference_lua/digest.rst

```diff
@@ -305,11 +306,12 @@ Below is a list of all ``digest`` functions.
 
     Returns a number made with consistent hash.
 
-    The guava function uses the `Consistent Hashing <https://en.wikipedia.org/wiki/Consistent_hashing>`_
-    algorithm of the Google
-    guava library. The first parameter should be a hash code; the second
-    parameter should be the number of buckets; the returned value will be an
-    integer between 0 and the number of buckets. For example,
+    The guava function uses the
+    `Consistent Hashing <https://en.wikipedia.org/wiki/Consistent_hashing>`_
+    algorithm of the Google guava library. The first parameter should be a hash
```

Comment:
Can we put an external link to the doc or any other reference about this library?

Reply:
noted it in https://github.com/tarantool/doc/issues/2019